roles/edit-role-binding.yaml

```diff
@@ -11,3 +11,54 @@ subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: Group
   name: ${DEVELOPERS_GROUP}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: developer-edit-custom-resources
+rules:
+  - apiGroups:
+      - 'keda.k8s.io'
+    resources:
+      - 'scaledobjects'
+    verbs:
+      - 'create'
+      - 'update'
+      - 'delete'
+      - 'get'
```

Comment:
add get and list to the view role as well?

Reply:
Ah remove from this one and add to view. Yeah sure

Reply:
I guess read roles need to be in both places. I think we apply one one of them for developers (edit in preview and read everywhere else)